examples/interaction/tools/hover_tooltip_advanced.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+
+from bokeh.models import HoverTool, Styles
+from bokeh.models.dom import Div, Index, ValueRef
+from bokeh.palettes import Spectral11
+from bokeh.plotting import figure, show
+
+N = 1000
+x = np.random.random(size=N) * 100
+y = np.random.random(size=N) * 100
+radii = np.random.random(size=N) * 1.5
+colors = np.random.choice(Spectral11, size=N)
+
+p = figure(
+    title="Demonstrates hover tool with advanced and regular tooltip formatting side-by-side",
+    tools="pan,wheel_zoom,box_select,crosshair",
+)
+
+p.circle(x, y, radius=radii, fill_color=colors, fill_alpha=0.6, line_color=None)
+
+grid = Div(
+    style=Styles(
+        display="grid",
+        grid_template_columns="auto auto",
+        column_gap="10px",
+    ),
+    children=[
+        "index:",  Div(children=["#", Index()]),
```

Comment:
Maybe add a style to this `Div`. 

I feel like this example shows the API but not why to use it, which is a bit sad.

Reply:
This is also a bit much. I will see if I can simplify this with `HTML()` model.

Reply:
The "improved" example is still quite much, but is shows styling and betters shows how to reference things.